.github/workflows/main.yml

```
@@ -60,6 +60,9 @@ jobs:
 
   settings:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.10"]
```

It may be worth adding some documentation about the black magic :wink: 

```suggestion
        // This will be used by the base setup action
        python-version: ["3.7", "3.10"]
```

---

Thanks!